Doc/library/sys.rst

```diff
@@ -1367,13 +1369,14 @@ always available.
 
 .. data:: version_info
 
-   A tuple containing the five components of the version number: *major*, *minor*,
-   *micro*, *releaselevel*, and *serial*.  All values except *releaselevel* are
-   integers; the release level is ``'alpha'``, ``'beta'``, ``'candidate'``, or
-   ``'final'``.  The ``version_info`` value corresponding to the Python version 2.0
-   is ``(2, 0, 0, 'final', 0)``.  The components can also be accessed by name,
-   so ``sys.version_info[0]`` is equivalent to ``sys.version_info.major``
-   and so on.
+   A :class:`~collections.namedtuple` containing the five components of the
+   version number: *major*, *minor*, *micro*, *releaselevel*, and *serial*.
+   All values except *releaselevel* are integers; the release level is
+   ``'alpha'``, ``'beta'``, ``'candidate'``, or ``'final'``.  The
+   ``version_info`` value corresponding to the Python version 3.6.3 is
+   ``(major=3, minor=6, micro=3, releaselevel='final', serial=0)``.  The
```

Comment:
`(major=3, minor=6, micro=3, releaselevel='final', serial=0)` looks too verbose. And this is not a valid Python syntax. If you want to expose `repr(sys.version_info)`, you should add `sys.version_info` at the start and make it yet more verbose.

The current writing looks better to me. This is a tuple of version_info components. It is common to compare  version_info with a tuple.
```python
if sys.version_info < (3, 7):
    ...
```